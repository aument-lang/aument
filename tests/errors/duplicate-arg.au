struct A{}
func (self: A) a(self) {}