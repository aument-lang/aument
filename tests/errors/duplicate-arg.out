parser error(13) in -: this argument is already declared
2 | func (self: A) a(self) {}
                     ^^^^
